x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -143,27 +147,28 @@ func GetCalcSetNetworkRegrets(
 		}
 	}
 
-	sort.Slice(networkLosses.ForecasterValues, func(i, j int) bool {
-		return networkLosses.ForecasterValues[i].Worker < networkLosses.ForecasterValues[j].Worker
+	sortedForecasterValues := args.NetworkLosses.ForecasterValues
+	sort.Slice(sortedForecasterValues, func(i, j int) bool {
+		return sortedForecasterValues[i].Worker < sortedForecasterValues[j].Worker
 	})
```

Comment:
already sorted?

Reply:
you are right sorted in synth.CalcNetworkLosses